platform/podjs/src/browserPod.ts

```diff
@@ -107,8 +169,86 @@ class FileUrl {
     ) {}
 }
 
-/* eslint-disable @typescript-eslint/no-unused-vars */
-class LocalStoragePolyOut implements PolyOut {
+interface FileInfo {
+    id: string;
+    name: string;
+    time: string;
```

Comment:
Why not a Date?

Reply:
1. That is how it was stored before
2. The `Stats` interface (as returned by `PolyIn.stat`) demands this format

Reply:
It does have a `getTime` string, but these need not be type-compatible. You can still convert it, gaining type safety. Is there any other reason? Even if it was not stored before that way, this is a refactoring, is there any reason why this interface should be backwards-compatible that way?

Reply:
It just seemed unrelated to the changes here. Also it adds complexity, converting the string to a `Date` object (in the migration code), just to convert it back to a string in `Stats.getTime()` but the gained type safety is a valid point. Done.